activation/licensed-users-count.md

```diff
@@ -0,0 +1,32 @@
+---
+description: This article explains how Microsoft 365 Licences work with SysKit Point.
+---
+
+# Licensed Users Count
+
+Using SysKit Point and all of its features, requires having licenses assigned to users in the Microsoft 365 tenant.
```

Comment:
Malo mi je neistinita ova izjava.
SysKit Point ne zahtijeva da imaš licence assignane userima u M365 tenantu. Assignane su userima kako bi oni mogli koristiti M365, i to već prije nego itko od customera pomisli na Point.
Naš licensing ovisi o M365 licencama kao što stoji u rečenicu ispod. 
Možda najbolje ovo maknuti.